kahuna/public/js/components/gr-top-bar/gr-top-bar.css

```diff
@@ -2,7 +2,7 @@ gr-top-bar {
     display: block;
     height: 50px; /* faking we're in the flow */
     position: relative;
-    z-index: 3;
+    z-index: 30;
```

Comment:
I was thinking of having layers, the same way you do with tabindexes - makes it easier to add things.

I was thinking you could have:
* 1 -> everything
* 10 -> things to be just above everything
* 20 -> user controls i.e. panels
* 30 -> the top layer i.e. topbar.